codegen/src/generator/rest_json.rs

```diff
@@ -115,6 +115,78 @@ impl GenerateProtocol for RestJsonGenerator {
     }
 }
 
+fn http_code_to_status_code(code: Option<i32>) -> String {
+    match code {
+        Some(actual_code) => {
+            // From http://hyper.rs/hyper/v0.9.12/src/hyper/src/status.rs.html and modified to return strings:
```

Comment:
You can get a string representation of an enum that derives Debug (hyper's `StatusCode` does) by implementing `fmt::Display` for it.  Here's an example - https://play.rust-lang.org/?gist=9cbf29a84aa396c38cb1499080ed4356&version=stable&backtrace=0